scripts/versionPlugin.js

```diff
@@ -0,0 +1,167 @@
+/**
+ * Versions the FaustWP plugin.
+ */
+
+const fs = require("fs");
+const path = require("path");
+const util = require("util");
+const readFile = (fileName) => util.promisify(fs.readFile)(fileName, "utf8");
```

Comment:
Since Node v11 you can get promisified versions of readFile and writeFile so you can use them instead of the util.promisify and it becomes a little bit neater:

https://nodejs.org/api/fs.html#fspromisesreadfilepath-options

const fs = require('fs/promises');
const path = require("path");

const readFile = (filename) => fs.readFile(filename, {encoding: "utf8"});
const writeFile = fs.writeFile;

Reply:
Ooo, nice tip! Changed in e40fa76.